### ISSUE TITLE
Support Internal&External LUT Texture 

### ISSUE THREAD

**Issue by hooh-hooah** [ore-/AI_Graphics#41]
As @ore- stated in #40, IBL needs to cover every single feature ILLUSION provided since we're taking another step further from ILLUSION's nest.

`abdata/studio/lut/00.unity3d` contains all the LUT information that post processing uses.

**Comment by ore-** [comment]
Having researched this quite a bit, my conclusion is: 
tl;dr - try [Amplify Colour's implementation](https://forum.unity.com/threads/free-amplify-color-advanced-color-grading.204253/)

According to [Thomas Hourdel](https://github.com/Chman), unity's postprocessing lead, post stack v2 LUT is LDR only, and there are no known 3d log LUTs required for HDR available for sale: https://github.com/Unity-Technologies/PostProcessing/issues/507

I don't think anyone would have the time to do this: https://github.com/Unity-Technologies/PostProcessing/wiki/(v2)-External-LUT-authoring

see other attempts at using strip LUTs and converting 3d cube LUTs for use in v2:
https://github.com/Philipp-Seifried/Unity-PostProV2-User-LUT-And-Z-Grading
https://forum.unity.com/threads/post-process-volume-color-grading-3d-lut-cube-format-incorrect-color-display.628828/

for now, I'll restrict the LUT option to LDR only since it doesn't do anything with HDR colour grading enabled.

**Comment by hooh-hooah** [comment]
I know it's good but to implement external LUT we need more examples people to follow.

**Comment by ore-** [comment]
That's not what I said at all. I'm saying we will NOT provide support for external LUT, not the one provided by unity PPv2, it's not worth it.

**Comment by ore-** [comment]
closing as 0.2.0 exposed builtin LUT and support for external 3D log LUT it's not worth the investment.
